requirements.txt

```diff
@@ -8,4 +8,4 @@ requests <3
 requests_ntlm <2
 lxml <5
 pyotp <3
-urllib3<1.25,>=1.20
```

Comment:
@xadamxk why was this removed?

Reply:
More specifically, why was the `<1.25` constraint removed?

Reply:
I was getting conflicts with boto3, since it has different requirements for urllib3, they were conflicting.

Reply:
Python 3.8 compatibility:
```
ERROR: botocore 1.19.6 has requirement urllib3<1.26,>=1.25.4; python_version != "3.4", but you'll have urllib3 1.24.3 which is incompatible.
```